lib/LedgerSMB/Entity/Location.pm

```diff
@@ -111,7 +111,7 @@ $self->set_class_name is called.
 
 our %classes = ( 1 => $locale->text('Billing'),
                  2 => $locale->text('Sales'),
-                 3 => $locale->text('Shipping'),
+                 3 => $locale->text('Shipping'),  ## no critic (ProhibitMagicNumbers) sniff
```

Comment:
Can't we create file-local constants for these? Or does that make no sense at all?

Reply:
I am confused.  I do not grasp the idea behind using ints as hash keys; it just seems like useless indirection to me.    'Physical' and 'Marketing" already seem to be missing.

Reply:
Where is `%classes` used?

Reply:
Where is `%classes` used?

Reply:
Nowhere, from what I can see. I've run the test-suite with it commented out and the tests still run fine...